bookwyrm/models/book.py

```diff
@@ -363,6 +365,16 @@ def save(self, *args, **kwargs):
             for author_id in self.authors.values_list("id", flat=True):
                 cache.delete(f"author-books-{author_id}")
 
+        # Create sort title by removing articles from title
+        if self.sort_title is None:
+            articles = chain(
+                *(LANGUAGE_ARTICLES[language] for language in self.languages)
+            )
+            icase_articles = (
+                f"[{a[0].capitalize()}{a[0].lower()}]{a[1:]}" for a in articles
```

Comment:
Would it be easier just to fully lowercase everything here, rather than trying to use title case?

Reply:
👍 I agree, especially since the `sort_title` will not be displayed on the site.

Reply:
I think a logical next step from here is to let users set custom sort titles in the edit book view, so it would show up there, but imo that's out of scope for this PR